lib/workers/repository/update/branch/execute-post-upgrade-commands.ts

```diff
@@ -63,12 +62,17 @@ export async function postUpgradeCommandsExecutor(
       }
 
       for (const cmd of commands) {
-        if (allowedCommands!.some((pattern) => regEx(pattern).test(cmd))) {
+        const compiledCmd = compile(cmd, mergeChildConfig(config, upgrade));
+        if (compiledCmd !== cmd) {
+          logger.debug(
+            { rawCmd: cmd, compiledCmd },
+            'Post-upgrade command has been compiled',
+          );
+        }
+        if (
+          allowedCommands!.some((pattern) => regEx(pattern).test(compiledCmd))
```

Comment:
```suggestion
          allowedCommands?.some((pattern) => regEx(pattern).test(compiledCmd))
```

Reply:
no, it's already validated here, but typescript didn't know it